Multisig/UI/Transaction/PaymentMethod/ChoosePaymentViewController.swift

```diff
@@ -75,7 +96,17 @@ class ChoosePaymentViewController: UIViewController, UITableViewDelegate, UITabl
     private func buildRelayerCell(tableView: UITableView) -> UITableViewCell {
         let cell = tableView.dequeueCell(PaymentMethodCell.self)
         cell.accessoryType = .none
-        cell.setRelaying(4, ReviewExecutionViewController.MAX_RELAY_TXS)
+        cell.setRelaying(remainingRelays, ReviewExecutionViewController.MAX_RELAY_TXS)
         return cell
     }
 }
+
+extension UITableViewCell {
```

Comment:
If the intention of this to make the cell not enabled then you can use: 
cell.isEnabled = false 
If the intention is to make it visually disabled then I think just setting subView.isEnabled = false will be enough no ?

Reply:
IIRC tried ` cell.isEnabled = false` but somehow it didn't work. Will try once more